lib/ingres_sa_dialect/ingresdbi.py

```diff
@@ -17,9 +18,14 @@ class Ingres_ingresdbi(IngresDialect):
     def __init__(self, **kwargs):
         IngresDialect.__init__(self, **kwargs)
 
-    @classmethod
-    def dbapi(cls):
-        return __import__('ingresdbi')
+    if (int((sqlalchemy.__version__).split('.')[0]) >= 2):
```

Comment:
same comment on compare - maybe but shared logic in the base rather than each implementation file?
create tuple in base (as a global) and then compare is simple:

    (1, 2, 3) > ......

e.g.

    sqlalchemy_version_tuple - tuple(map(int, sqlalchemy.__version__.split('.')))  # assuming ther isn't one already we can make use of

Reply:
Now using tuple variable defined in base.py.